incubator/elasticsearch/templates/clusterrole.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.rbac.create -}}
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Note, please see comment in https://github.com/kubernetes/charts/pull/2544#pullrequestreview-71194400 about k8s version support.